ui-library-filesharing-chat-composite/app/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-calling": "1.16.3-beta.1",
+    "@azure/communication-calling": "1.17.1-beta.5",
     "@azure/communication-chat": "1.4.0-beta.1",
```

Comment:
The new beta for chat is out. Not sure if we should also do this bump. https://www.npmjs.com/package/@azure/communication-chat/v/1.4.0-beta.2

Reply:
No, as this is a breaking change and not part of the ui library beta release.